### ISSUE TITLE
LiveReload not working with Sublime text 3 in Linux (?)

### ISSUE THREAD

**Issue by thanhngvpt** [spf13/hugo#2522]
Hi author.
I want to use live reload in hugo go, i've read https://gohugo.io/extras/livereload/ but i dont know where to put </body> (is it in theme?). For more detail i expect whenever i save the post that i am editing browser will auto refresh to display newest data (hugo server auto restart or anything else..)
Many thanks

**Comment by bep** [comment]
* What version of Hugo, what operating system?
* Is there anything special with the disk you store files? Network disk?
* Do you see anything in the console when you perform a change?

**Comment by thanhngvpt** [comment]
@bep My system info:
- My Hugo: Hugo Static Site Generator v0.17-DEV BuildDate: 2016-10-03T10:05:07+07:00
- OS: Archlinux
- Disk and network in normal.
- Console log nothing change

**Comment by thanhngvpt** [comment]
There are some infomations when i use visual studio code to edit file, livereload work well, but if i use sublime text 3 to edit file livereload not working.

**Comment by bep** [comment]
OK, I did some work in https://github.com/fsnotify/fsnotify for OSX re. atomic saves in editors, but there may also bee some Linux editors with trouble.

Assuming you build Hugo yourself, could you do a `go get -u -v` in Hugo and rebuild it with `make install` -- and try again with sublime text and report back with the output of `hugo config`.

**Comment by anthonyfok** [comment]
Hi @ bep, thank you for the /cc.

@thanhngvpt, sorry, I haven't been able to reproduce the problem that you reported.  I have just tested Sublime Text 3, stable channel, Build 3126 (released 23 September 2016), on Debian sid (unstable), with both Hugo v0.16 and the just-released Hugo v0.17, and upon pressing Ctrl+S to save a `content/post/somefile.md` that I was editing, `hugo server` automatically detects it and rebuild the site, e.g.:

```
Change detected, rebuilding site
2016-10-09 20:19 -0600
Source changed /home/foka/snappy-hugo/content/post/Dependencies.md
Built site for language en:
0 of 1 draft rendered
0 future content
0 expired content
6 pages created
0 non-page files copied
0 paginator pages created
0 tags created
0 categories created
total in 31 ms
```

My `/home` partition is formatted as an ext4 file system, and my running kernel is:

* Linux debian-Aspire-M5-481G 4.7.0-1-amd64 #1 SMP Debian 4.7.5-1 (2016-09-26) x86_64 GNU/Linux
